scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   // below the smallest value in the build output; this avoids the
   // need for frequent changes as bundle sizes fluctuate.
   MIN_SIZE: 673,
-  MAX_SIZE: 851,
+  MAX_SIZE: 857,
```

Comment:
New size was 852. 857 follows our standard of +/- 5

Reply:
Did `MIN_SIZE` also change?